tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -100,30 +92,41 @@ class WskBasicCliUsageTests
 
     it should "show api build version using property file" in {
         val tmpwskprops = File.createTempFile("wskprops", ".tmp")
-        val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
-        wsk.cli(wskprops.overrides ++ Seq("property", "set"), env = env)
-        val stdout = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env).stdout
         try {
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            wsk.cli(Seq("property", "set", "-i") ++ wskprops.overrides, env = env)
+            val stdout = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env).stdout
             stdout should include regex ("""(?i)whisk API build\s+201.*""")
         } finally {
             tmpwskprops.delete()
         }
     }
 
     it should "fail to show api build when setting apihost to bogus value" in {
-        val wsk = new Wsk(usePythonCLI = true)
-        val rr = wsk.cli(Seq("--apihost", "xxxx.yyyy", "property", "get", "--apibuild"), expectedExitCode = ANY_ERROR_EXIT)
-        rr.stdout should { include regex ("Cannot determine API build") or include regex ("Unknown") }
-        rr.stdout should not include regex("""(?i)whisk API build\s+201.*""")
-        rr.stderr should not include ("https:///api/v1: http: no Host in request URL")
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        try {
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            wsk.cli(Seq("property", "set", "-i", "--apihost", "xxxx.yyyy"), env = env)
+            val rr = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env, expectedExitCode = 3)
+            rr.stdout should include regex ("""whisk API build\s*Unknown""")
+            rr.stderr should include regex ("Unable to obtain API build information")
+        } finally {
+            tmpwskprops.delete()
+        }
     }
 
     it should "show api build using http apihost" in {
-        val wsk = new Wsk(usePythonCLI = true)
-        val apihost = s"http://${WhiskProperties.getControllerHost}:${WhiskProperties.getControllerPort}"
-        val stdout = wsk.cli(Seq("--apihost", apihost, "property", "get", "--apibuild")).stdout
-        stdout should not { include regex ("Cannot determine API build") or include regex ("Unknown") }
-        stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        try {
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            val apihost = s"http://${WhiskProperties.getControllerHost}:${WhiskProperties.getControllerPort}"
+            wsk.cli(Seq("property", "set", "--apihost", apihost), env = env)
+            val rr = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env, expectedExitCode = 3)
```

Comment:
why is this right?  line 123 sets the api host shouldn't 124 retrieve the build info?